browser/src/map/handler/Map.Keyboard.js

```diff
@@ -770,6 +770,10 @@ L.Map.Keyboard = L.Handler.extend({
 						else {
 							partToSelect = currentSelectedPart != 0 ? currentSelectedPart - 1 : parts - 1;
 						}
+
+						if (this._map._docLayer.isAutoFillPopupOpen)
+							this._map.fire('closeautofillpopup');
```

Comment:
please just fire an event and do the "if ( ... ) " in the handler

Reply:
this is removed in the new commits: no need